tutorials/migration.md

```diff
@@ -4,96 +4,846 @@ layout: intro
+lastUpdate). Restart openHAB and verify those Items configured with 
+restoreOnStartup are initialized to their last state.
+- **Rules**: When you first copy over your rules there will be many errors, 
+particularly if the rules assume that all Items have a state. One can easily 
+forget to check for NULL in an openHAB instance that has been running for some
+time and one is using restoreOnStartup. But because this is a brand new instance
+there are no values in the database to restore the Items to when openHAB starts.
+- **Sitemap**: The most 
+common Sitemap issue will be missing icons. Browse through your sitemap 
+methodically and identify those entries that have a missing or wrong
+icon. Select an alternative from the [defaults](../addons/iconsets/classic/readme.html)
+or copy the one you were using from openHAB 1.x to the conf/icons/classic folder.
+Both BasicUI and ClassicUI pull their icons from that folder. For a sitemap that
+most closely resembles your current sitemap I recommend using ClassicUI.
 
-### Understanding "Things"
+Once you are satisfied that your new openHAB system is up and running take a 
+deep breath and take a break. Let it run for a few days or a week and verify 
+that everything is working as it should. Watch the logs for inexplicable errors.
+Once you are happy with how it is running you can start migrating to the openHAB
+2 native bindings and start taking advantage of Things and Channels (see the 
+next section).
 
-One of the new notions that people quickly come across when moving to openHAB 2 is that of a ["Thing"](../concepts/things.html).
-To understand what they are and how they relate to Items, we have to look at the past:
+Now is also a good time to install [Eclipse SmartHome Designer](https://www.eclipse.org/smarthome/documentation/community/downloads.html)
+to edit and review your files. 
 
-In openHAB 1 you had to add a so-called ["binding configuration"](https://github.com/openhab/openhab/wiki/Explanation-of-items#binding-config) in curly brackets to your item definition in order to link your item (representing a logical function in your setup) to some physical device. The openHAB wiki gives a nice example of how that looked like:
 
-```
+## New Concepts: Things and Channels
+
+One of the new notions that people quickly come across when moving to openHAB 2 
+native bindings is that of a ["Thing"](../concepts/things.html). To understand 
+what they are and how they relate to Items, we will compare them to how Items 
+are "bound" and configured in openHAB 1.x.
+
+In openHAB 1.x (and importantly still for 1.9 version bindings running in openHAB
+2) one had to add 
+["binding configuration"](https://github.com/openhab/openhab/wiki/Explanation-of-items#binding-config) 
+in curly brackets to the end of Item definitions in order to link that Item to 
+some control point on a physical device or API. The 
+[openHAB wiki](https://github.com/openhab/openhab/wiki) gives a nice examples of 
+how this looks:
+
+~~~~
```

Comment:
Again, wouldn't \`\`\` be more appropriate? I won't comment on further uses of this.